hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestMultiRespectsLimits.java

```diff
@@ -150,10 +150,10 @@ public void testBlockMultiLimits() throws Exception {
       Bytes.toBytes("5"), // Buffer
     };
 
-    // Set the value size so that one result will be less than the MAX_SIE
+    // Set the value size so that one result will be less than the MAX_SIZE
     // however the block being reference will be larger than MAX_SIZE.
     // This should cause the regionserver to try and send a result immediately.
-    byte[] value = new byte[MAX_SIZE - 100];
+    byte[] value = new byte[MAX_SIZE / 2];
```

Comment:
I wonder whether / 2 is enough? If it is too big, then testBlockMultiLimits will be the same with testMultiLimits?